### PR TITLE
Adds /account/balance endpoint

### PR DESCRIPTION
**Pull request by LindaOrtega**
- [x] blocker: have /local return the block hash and block height of the block where the local command was run in. Currently, it returns the parents of the block and (+1 of the block height).
- [ ] ~~Research possible bug (checkpointer error keeps being thrown when I hit this endpoint in non-genesis case)~~ Can't replicate and it seems to work fine with a local mining node now.

### CODE REVIEW

File: src/Chainweb/Rosetta/RestAPI.hs [hunk 1]
```diff
@@ -118,9 +137,24 @@ validateNetwork v (NetworkId bc n msni) = do
...
   mkChainId v cid
+
+
+kdaToRosettaAmount :: Decimal -> Amount
+kdaToRosettaAmount k = Amount (sshow amount) currency Nothing
+  where
+    -- Value of the transaction in atomic units represented as an
+    -- arbitrary-sized signed integer.
+    amount :: Integer
+    amount = floor $ k * (realToFrac $ (10 :: Integer) ^ numDecimals)
```
Comment (LindaOrtega) [inline src/Chainweb/Rosetta/RestAPI.hs:155]: Is this the correct approach?

File: src/Chainweb/Rosetta/RestAPI.hs [hunk 2]
```diff
@@ -118,9 +137,23 @@ validateNetwork v (NetworkId bc n msni) = do
...
+
 -- | Guarantees that the `ChainId` given actually belongs to this
 -- `ChainwebVersion`.
 readChainIdText :: ChainwebVersion -> T.Text -> Maybe ChainId
 readChainIdText v c = do
   cid <- readMaybe @Word (T.unpack c)
   mkChainId v cid
+
+
+kdaToRosettaAmount :: Decimal -> Amount
```
Comment (slpopejoy) [inline src/Chainweb/Rosetta/RestAPI.hs:149]: Suggest a quick unit test for just this function

Comment (LindaOrtega) [reply src/Chainweb/Rosetta/RestAPI.hs:149]: I merged this PR into the #1050 and added a unit test for `kdaToRosettaAmount` there

**Comment by LindaOrtega** [comment]
Closing this as it was merged into #1050

**Pull request closed without merging**